scripts/removeBoundingBox.js

```diff
@@ -0,0 +1,35 @@
+// from the root folder, run `node ./scripts/removeBoundingBox.js`
+
```

Comment:
Can we add this as a script in package.json and automatically run it after we create the /svg folder?